kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -347,19 +341,11 @@
     margin-right: auto
     width: ($iphone-5-width - 20)px
 
-  .terms
-    background-color: $core-bg-light
-    color: $core-text-annotation
-    height: 6em
-    overflow-y: scroll
-    padding: 0.5em
-    margin-bottom: 1em
-    p
-      margin-top: 0
-
   .app-bar-icon
-    font-size: 2.5em
-    margin-left: 0.25em
+    display: inline-block
```

Comment:
Since we are using the ui-icon component, we do not need the `display: inline-block` since it is already inline-block. Also, ui-icons are sized by changing the font-size. So something like the following should get the job done.

```css
.app-bar-icon {
    margin-left: 16px;
    font-size: 32px;
}
```

> UiIcon shows an icon glyph. Icons are 24px by 24px (relative to a root font size of 16px), but the size can be changed by overriding the CSS font-size property of .ui-icon.

Reply:
We aren't going to be using a `ui-icon` in the final version, though. It looks like these styles are actually for the `<img>` tag containing the vf logo that's in the port. Maybe I should move the icon swap to `styles` rather than `port`?

Reply:
are we swapping the logo in network schools @jeepurs?

Reply:
> are we swapping the logo in network schools

I'm pretty sure we will also have vf colors and logos.

Reply:
swapped!